Sources/Generator/StyleGenerator.swift

```diff
@@ -30,23 +30,41 @@ public class StyleGenerator: Generator {
                 l(ifSimulator("import ReactantLiveUI"))
             }
         }
+        let styleImports = Set(group.styles.flatMap { $0.imports })
+        for styleImport in styleImports {
+            l("import \(styleImport)")
+        }
         l()
         l("struct \(group.swiftName)") {
             for style in group.styles {
                 l("static func \(style.styleName)(_ view: \(Element.elementMapping[style.type]?.runtimeType ?? "UIView"))") {
                     for extendedStyle in style.extend {
-                        guard let styleName = extendedStyle.components(separatedBy: ":").last else {
+                        let components = extendedStyle.components(separatedBy: ":").filter { $0.isEmpty == false }
+                        if let styleName = components.last {
+                            if let groupName = components.first, components.count > 1 {
+                                l("\(groupName.capitalizingFirstLetter() + "Styles").\(styleName)(view)")
+                            } else {
+                                l("\(group.swiftName).\(styleName)(view)")
+                            }
+                        } else {
                             continue
                         }
-                        l("\(group.swiftName).\(styleName)(view)")
                     }
                     for property in style.properties {
                         l(property.application(on: "view"))
                     }
                 }
             }
         }
-        
+
         return output
     }
 }
+
+extension String {
+    public func capitalizingFirstLetter() -> String {
```

Comment:
`private`/`fileprivate`?

Reply:
I'll rather move it to a new file.

Reply:
I don't feel like it should be a public function populating user's index.

Reply:
Generator is not accessible by user and without modifier it should be available only in the module, shouldn't it?